web/src/components/ProjectStakeholders/__snapshots__/ProjectStakeholders.test.tsx.snap

```diff
@@ -0,0 +1,11 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Stakeholders should render legacy stakeholder items 1`] = `
+Array [
+  <ul
+    aria-label="Legacy stakeholders"
+  >
+    <li />
```

Comment:
Not sure this snapshot is what you're after?

Reply:
I will take a look